api/src/main/java/org/pmiops/workbench/billing/BillingProjectBufferService.java

```diff
@@ -163,6 +162,32 @@ public void cleanBillingBuffer() {
             });
   }
 
+  private List<DbBillingProjectBufferEntry> findExpiredCreatingEntries(Instant now) {
+    return StreamSupport.stream(
+            billingProjectBufferEntryDao
+                .findAllByStatusAndLastStatusChangedTimeLessThan(
+                    DbStorageEnums.billingProjectBufferStatusToStorage(CREATING),
+                    new Timestamp(
+                        now.minus(CREATING_TIMEOUT_MINUTES, ChronoUnit.MINUTES).toEpochMilli()))
+                .spliterator(),
```

Comment:
IntelliJ tells me you can just do 
```
  private List<DbBillingProjectBufferEntry> findExpiredCreatingEntries(Instant now) {
    return billingProjectBufferEntryDao
        .findAllByStatusAndLastStatusChangedTimeLessThan(
            DbStorageEnums.billingProjectBufferStatusToStorage(CREATING),
            new Timestamp(
                now.minus(CREATING_TIMEOUT_MINUTES, ChronoUnit.MINUTES).toEpochMilli()))
        .stream()
// ...
```

Reply:
also nit: I'd probably declare the cutoff timestamp separately, since it's a  slightly involved construction and calculation.